src/formatDate.js

```diff
@@ -50,7 +50,30 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newFormat = [];
+  const dateArr = date.split(fromFormat[3]);
+  const dateYear = dateArr[fromFormat.indexOf('YYYY')]
+    || dateArr[fromFormat.indexOf('YY')];
+
+  for (let i = 0; i < toFormat.length - 1; i++) {
```

Comment:
Consider using a 'for...of' loop instead of a 'for' loop to iterate over the 'toFormat' array. This makes the code more readable and avoids the need to use an index. For example: `for (const format of toFormat) { ... }`